<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>showandsort</class>
 <widget class="QWidget" name="showandsort">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>639</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>展示（排序）界面</string>
  </property>
  <widget class="QPushButton" name="returnpushButton">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>480</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>主菜单</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="nameradioButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>115</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>姓名</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="numberradioButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>90</y>
     <width>115</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>手机号码</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sortpushButton">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>80</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>排序</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>571</width>
     <height>321</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
